contents/blog/raising-3m-for-os.md

```diff
@@ -0,0 +1,230 @@
+---
+date: 2020-06-02
+title: How we raised $3M
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+Open source projects have long battled with how to finance themselves. We're lucky to have significant funding and wanted to share what we did to help other cool projects take off.
+
+For those who've not met us before, PostHog provide open source product analytics. We went through the YCombinator [W20 batch](https://www.ycombinator.com/companies?batch=w2020), which took place from January to March 2020. We have raised $3.025M since we started in January, and wanted to share what we did. We may well not be a typical company, but we would hope this gives some lessons learned regardless.
```

Comment:
This leaves ambiguous whether we raised this money since we started the company, or since we started fundraising. i think the theme of the article should be 'we've done this bloody quickly after starting', though that might make us look inexperienced...